Dockerfile

```diff
@@ -15,6 +15,8 @@
 ARG PYT_VER=23.06
 FROM nvcr.io/nvidia/pytorch:$PYT_VER-py3 as builder
 
+ARG TARGETPLATFORM
```

Comment:
Should we default this to `linux/amd64` to simplify our predominant build command?

Reply:
I have tried this. The issue is, the default arg will work with `docker build` command. But with `docker buildx`, the default specified in the Dockerfile overrides the arg passed via the `--platform` command line arg. Thus, `docker buildx` will fail for say `arm64`.